packages/firebase_admob/example/pubspec.yaml

```diff
@@ -0,0 +1,53 @@
+name: firebase_admob_example
+description: Demonstrates how to use the firebase_admob plugin.
+
+dependencies:
+  flutter:
+    sdk: flutter
+  firebase_admob:
+    path: ../
+
+# For information on the generic Dart part of this file, see the
```

Comment:
remove (all) the generated boilerplate comments in this file? They don't really help to understand the plugin.